content/docs/studio/how-it-works.md

```diff
@@ -1,48 +1,54 @@
 # How Iterative Studio Works
 
-[Iterative Studio](https://studio.iterative.ai/) works with the data, metrics,
-hyperparameters and model metadata that you add to your ML project Git
-repositories. It works very closely with your Git ecosystem.
+[Iterative Studio](https://studio.iterative.ai/) works very closely with your
+Git ecosystem. It parses the data, metrics, hyperparameters and model metadata
+that you add to your ML project Git repositories, and renders them on a web UI.
+It also parses and displays live metrics and plots sent by [DVCLive]; these
+don't need to be pushed to the Git repositories.
 
-This video illustrates how Iterative Studio works closely with your Git
-ecosystem.
+The following video illustrate how Iterative Studio works with data from your ML
+projects and experiments.
 
 > Note that we have renamed DVC Studio to Iterative Studio and Views to
 > Projects.
 
 https://www.youtube.com/watch?v=5xM5az78Lrg
 
-## How your project data is saved
+## How you save your project data
 
 - Using [DVC] and [Git], you will push all your ML experiments to your GitHub,
   GitLab or Bitbucket repositories as Git commits.
 - Using Iterative Studio, or using the command line interface (CLI) of [GTO],
   and possibly [MLEM], you will push all your ML model details to the Git
   repositories as Git commits and Git tags.
+- Using [DVCLive], you will push live metrics and plots from your Git-committed
+  experiments to Iterative Studio.
```

Comment:
💅🏼 minor (just a bit shorter)
```suggestion
- Using [DVCLive], you will push live metrics and plots directly to Iterative
  Studio.
```

Reply:
I think it is important to mention that the experiments are committed to Git, coz we don't support live metrics for local experiments yet.

Reply:
> from your experiments running on a remote machine.

This is still confusing. We need the experiments to have been submitted from Studio and committed to Git. `Remote machine` could be any remote machine - for me, if I do some work in a cloud instance, then I'm working on a remote machine coz it's not my local machine.

I've updated the text, but differently as shown below:

`Using [DVCLive], you will push live metrics and plots from your experiments to Iterative Studio. Note that at the moment, live metrics are supported only for experiments that you [submit from the Iterative Studio UI].`

Let me know if it still doesn't make sense.

Reply:
It makes more sense now, thanks!